guacamole-jetty/src/main/java/dev/crafting/guacamole/Parameters.java

```diff
@@ -23,8 +34,16 @@ public class Parameters {
     private static Parameters _shared = new Parameters();
 
     private Parameters() {
-        protocol = "rdp";
-        hostname = "localhost";
+        Properties properties = new Properties();
+        try (FileInputStream fis = new FileInputStream(Parameters.configFile);) {
+            properties.load(fis);
+        } catch (IOException e) {
+            System.err.println(e.getMessage());
+        }
+        protocol = properties.getProperty("protocol", "rdp");
+        hostname = properties.getProperty("hostname", "localhost");
+        username = properties.getProperty("username", "Administrator");
```

Comment:
Please do not add a default username. It's possible some RDP server doesn't need auth so no username/password in this case.